examples/azureopenai-private-endpoints/main.tf

```diff
@@ -23,6 +23,38 @@ module "openai" {
       scale_type    = "Standard"
     }
   }
+  diagnostic_settings = {
+     "aoai-svc-diag" = {
+      name = "aoai-svc-diag"
+      log_analytics_workspace_id = "/subscriptions/xxxxxxxxxxxxxxxx/resourceGroups/monitoring-rg/providers/Microsoft.OperationalInsights/workspaces/workspace-xxxxxxxxxxxxxxxxxxxxxxxxxx"
+      log_analytics_destination_type = "Dedicated"
+
+      audit_log_retention_policy = {
+        enabled = true
+        days = 30
+      }
+      request_response_log_retention_policy = {
+        enabled = false
+        days = 30
+      }
+      trace_log_retention_policy = {
+        enabled = false
+        days = 30
+      }
+      metric_retention_policy = {
+        enabled = false
+        days = 30
+      }
+    } 
+  }
+  network_acls = [{
+      default_action = "Deny",
+      ip_rules       = ["00.00.00.00"]
+      virtual_network_rules = [{
+        subnet_id                            = "/subscriptions/xxxxxxxxxxxxxxxxxxxxxxxxxxxxx/resourceGroups/vnet-rg/providers/Microsoft.Network/virtualNetworks/vnet-main/subnets/vnet-main"
```

Comment:
Same here, provide the real value. Get the subnet id from the module vnet in the `network.tf` file.

Reply:
done. thank you for the pointers